pegasus/sites.v3/code.org/public/about/careers/engineering.haml

```diff
@@ -0,0 +1,82 @@
+---
+title: Join Our Engineering Team!
+theme: responsive_full_width
+---
+
+%link{href: "/css/generated/design-system-pegasus.css", rel: "stylesheet"}
+%link{href: "/css/generated/page/engineering-styles.css", rel: "stylesheet"}
+
+%section.hero-banner-basic.hero-banner-basic--engineering
+    .wrapper
+        .flex-container
+            .text-wrapper.col-60.flex-container.align-items-center
+                .hero-banner-basic__content
+                    %h1{style: "color: white"}
+                        =hoc_s(:engineering_hero_heading)
+                    %p{style: "color: white"}
+                        =hoc_s(:engineering_hero_desc)
+                    %a{href: "https://code.org/about/careers#roles ", class: "link-button", target: "_blank", rel: "noopener noreferrer", aria:{label: hoc_s("accessibility_explore_title")}}
+                        =hoc_s(:call_to_action_open_positions)
+            .col-40.flex-container.justify-end
+                %img.hero-banner-basic__image{src: "/images/about/careers/engineering/hero-image.png", alt: ""}
+        .clear
+
+
+%section.engineering-team#accessibility-team
```

Comment:
Is this class `#accessibility-team` being used for anything?

Reply:
nope =) 
removed